components/UI/featured_banner/featuredQuest.tsx

```diff
@@ -62,19 +63,10 @@ const FeaturedQuest: FunctionComponent<FeaturedQuestProps> = ({
           </>
           }
         <div className="flex items-center mb-4 mt-6 gap-2">
-          {issuer?.name || issuer?.logoFavicon ? (
-            <div className={styles.issuer}>
-              <CDNImage
-                alt={"Feature Quest Reward"}
-                height={20}
-                width={20}
-                src={issuer?.logoFavicon}
-                className={styles.featuredQuestRewardIcon}
-              />
-              <p className={styles.featuredQuestReward}>{reward}</p>
-            </div>
-          ) : null}
```

Comment:
please replace the div tag inside the braces here. I think removing null states is not required